package/yast2-dhcp-server.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Tue Jan 24 10:52:56 CET 2017 - schubi@suse.de
+
+- AutoYaST: Fixed a crash while cloning the system.
```

Comment:
IMHO this would better describe it (it is not AY specific):
```
- Fixed a crash at start when the "dhcp-server" package was not
  installed in the system, fixes also the AutoYaST cloning
  (bsc#1021635).
```